frontend/app/src/entities/schema/ui/attribute-display.tsx

```diff
@@ -86,3 +83,23 @@ const ChoicesRow = ({
     </div>
   );
 };
+
+const AttributeParameters = ({
+  attribute,
+}: { attribute: components["schemas"]["AttributeSchema-Output"] }) => {
+  if (attribute.kind === "Text") {
+    return (
+      <div>
+        <PropertyTitle title="Parameters" />
+
+        <div className="pl-4">
+          <PropertyRow title="Regex" value={attribute.parameters.regex} />
+          <PropertyRow title="Min length" value={attribute.parameters.min_length} />
+          <PropertyRow title="Max length" value={attribute.parameters.max_length} />
+        </div>
+      </div>
+    );
+  }
```

Comment:
A thought around this, since we'll have parameters on a number of types would it make sense to have this part dynamically created? Perhaps with a mapping where if attribute.kind === "Text" we know what AttributeSchema type to expect and show all of the fields for that particular schema? That way if we add more fields later we don't need to keep it up to date on both sides? I think we can set a title for these within the schema as well.